doc/api/process.md

```diff
@@ -508,6 +508,16 @@ $ bash -c 'exec -a customArgv0 ./node'
 'customArgv0'
 ```
 
+## process.channel
+<!-- YAML
+added: REPLACEME
+-->
+
```

Comment:
Yes, I noticed that the style of the docs in this file are slightly different. I think a good first contribution would be to bring additional consistency to this file. For example, see https://nodejs.org/api/process.html#process_process_connected vs. https://nodejs.org/api/child_process.html#child_process_child_connected.